src/schemas/json/partial-pytest.json

```diff
@@ -36,33 +51,47 @@
               "type": "string"
             }
           ],
-          "description": "Extra command line options to be added by default.",
+          "description": "Add the specified OPTS to the set of command line arguments as if they had been specified by the user.",
           "x-tombi-array-values-order": "ascending"
         },
         "cache_dir": {
           "type": "string",
-          "description": "Sets directory for cache plugin. Can include environment variables.",
+          "description": "Sets the directory where the cache plugin's content is stored.",
           "default": ".pytest_cache"
         },
+        "collect_imported_tests": {
+          "type": "string",
+          "enum": ["true", "false"],
+          "description": "Settings this to false will make pytest collect classes/functions from test files _only_ if they are defined in that file (as opposed to imported there).",
+          "default": "true"
+        },
         "consider_namespace_packages": {
-          "type": "boolean",
```

Comment:
There were a number of these value with incorrect types. As they document in their [changelog](https://docs.pytest.org/en/stable/changelog.html#pytest-9-0-0-2025-11-05):

> While pytest, since version 6, supports configuration in pyproject.toml files under [tool.pytest.ini_options], it does so in an “INI compatibility mode”, where all configuration values are treated as strings or list of strings. Now, pytest supports the native TOML data model.

I understand this to mean that any non-string types to be technically incorrect, and could result in unexpected behaviours, hence why I have restricted the schema here to ensure nothing unexpected happens.